client/my-sites/stats/components/line-chart/index.tsx

```diff
@@ -38,57 +33,74 @@ function StatsLineChart( {
 		? formatTimeTick
 		: ( value: number ) => {
 				const date = new Date( value );
-				return new Date( date ).toLocaleTimeString( moment.locale(), {
-					hour: '2-digit',
-					minute: '2-digit',
-					hour12: true,
+				return date.toLocaleDateString( undefined, {
+					month: 'short',
+					day: 'numeric',
 				} );
 		  };
 
 	const formatViews = ( value: number ) => {
 		return value.toFixed( 0 ).toString();
 	};
 
-	const dataSeries = chartData?.[ 0 ].data || [];
+	const isEmpty = ( chartData?.[ 0 ].data || [] ).length === 0;
+
+	const maxViews = useMemo(
```

Comment:
We should probably rename all the variables with name `views` as it doesn't necessarily to be views - they could be anything

Reply:
yeah sounds like a good idea. I can tackle that in another PR so this one doesn't end up too massive.